examples/analog_out/cont_gen_voltage_wfm_int_clk_every_n_samples_event.py

```diff
@@ -0,0 +1,40 @@
+"""Example of analog output voltage generation with events.
+
+This example demonstrates how to use a Every N Samples events to output a
+continuous periodic waveform to an Analog Output Channel
+using an internal sample clock. The Every N Samples events indicate when
+the specified number of samples generation is complete.
+"""
+
+import numpy as np
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType
+
+with nidaqmx.Task() as task:
+    frequency = 10
+    amplitude = 1
+    sampling_rate = 100
+    duration = 1
+
+    # generate the time array
+    t = np.arange(0, duration, 1 / sampling_rate)
+    # Generate the sine wave
+    data = amplitude * np.sin(2 * np.pi * frequency * t)
+
+    def callback(task_handle, every_n_samples_event_type, number_of_samples, callback_data):
+        """Callback function for written data."""
+        print(f"transferred {number_of_samples} samples event invoked.")
```

Comment:
This is a run-on of two separate sentences:
- transferred N samples
- every N samples event invoked

Choose one or the other. (I like "Transferred N samples" better")

Reply:
This was a comment about the text. I wasn't suggesting to remove the f-string. Sorry if that wasn't clear.